compiler/rustc_llvm/src/lib.rs

```diff
@@ -2,42 +2,81 @@
 #![allow(internal_features)]
 #![doc(html_root_url = "https://doc.rust-lang.org/nightly/nightly-rustc/")]
 #![doc(rust_logo)]
+#![feature(extern_types)]
 #![feature(rustdoc_internals)]
 #![warn(unreachable_pub)]
 // tidy-alphabetical-end
 
-// NOTE: This crate only exists to allow linking on mingw targets.
```

Comment:
I'm not sure what this was originally referring to, but it's pretty clearly no longer true of the present crate.